### DISCUSSION TITLE
Windows 11 Duplicate Icon

### DISCUSSION THREAD

**Discussion by Turtle-Boi** [File-New-Project/EarTrumpet#973] [Q&A]
#914 
So now I just can't see the duplicate for some reason.

**Comment by Turtle-Boi** [comment]
I am on the preview version of Windows 11 if that matters.

**Reply by riverar** [reply]
Does it appear in the overflow? 
![image](https://user-images.githubusercontent.com/475132/146322052-f28518ac-5627-4800-b4c9-7fdf19946c4d.png)

Is it running? (Check Task Manager)
